File: mRemoteV1/CS/Connection/Connection.Protocol.Telnet.cs
using System.Collections.Generic;
using System;
using AxWFICALib;
using System.Drawing;
using System.Diagnostics;
using System.Data;
using AxMSTSCLib;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;


namespace mRemoteNG.Connection.Protocol
{
	public class Telnet : PuttyBase
	{
				
		public Telnet()
		{
			this.PuttyProtocol = Putty_Protocol.telnet;
		}
				
		public enum Defaults
		{
			Port = 23
		}
	}
}

File: mRemoteV1/Connection/Connection.Protocol.Telnet.vb
Namespace Connection

    Namespace Protocol
        Public Class Telnet
            Inherits PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.telnet
            End Sub

            Public Enum Defaults
                None = 0
                Port = 23
            End Enum
        End Class
    End Namespace

End Namespace